src/content/docs/en/developers/guides/scroll-messenger-cross-chain-interaction.mdx

```diff
@@ -84,8 +84,8 @@ We pass the message by executing `executeFunctionCrosschain` and passing the fol
 - `value`: In this case, it is `0` because the `setGreeting`is not payable.
 - `greeting`: This is the parameter that will be sent through the message. Try passing `“This message was crosschain!”`
 - `gasLimit`:
-  - If you are sending the message from L1 to L2, around `5000` gas limit should be more than enough.
-  - If you are sending the message from L2 to L1, pass `0`, as the transaction be completed by executing an additional transaction on L1.
+  - If you are sending the message from L1 to L2, around `5000` gas limit should be more than enough. That said, if you set this too high, any unused portion of the fee will be refunded.
```

Comment:
The gas limit is not refunded. The `gasLimit` * `baseFee` is the fee that is charged, and if the `msg.value` is higher than this fee, the remainder of the `msg.value` is refunded. Conversely, if the `msg.value` is lower than this fee, the tx will revert

Reply:
Fixed!